### PR TITLE
Fixing a bug and doing some cleanup

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/2156

Also did some general cleanup and added a tiny bit of info in the faq about Impeller.

cc @goderbauer (for pixel info -- should I add more?)
cc @chinmaygarde (for impeller info)

I plan to do a pass over the website and, wherever Skia is mentioned, evaluate whether Impeller also needs mentioning.

### CODE REVIEW

File: src/resources/faq.md [hunk 1]
```diff
@@ -131,22 +147,33 @@ Flutter includes:
...
+[Android Studio][], and [IntelliJ IDEA][].
+See [editor configuration][] for setup details,
+and [VS Code][] and [Android Studio/IntelliJ][]
 for tips on how to use the plugins.
 
 Alternatively, you can use the `flutter` command
 from a terminal, along with one
 of the many editors that support [editing Dart][].
 
+
```
Comment (goderbauer) [inline src/resources/faq.md:169]: nitty-nit: throughout this document, the list of links is sometimes separated by two blank lines, sometimes by one. Not sure if you want to follow a unified style here?

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by parlough — APPROVED** [review]
Love the cleanup and updates!

File: src/resources/faq.md (hunk 1 above)
Comment (sfshaza2) [reply src/resources/faq.md:169]: LOL. Yeah, if there is only one link, I tend to use one line. If there are multiple links, I tend to use two. I am a bit conflicted about it.  :)

**Pull request merged**